### PR TITLE
Homework 7

### PR DESCRIPTION
**Pull request by shivakcpm**
* Add formik library
* Used formik hooks for addmovie component

### CODE REVIEW

File: src/components/AddMovie/AddMovieComponent.js [hunk 1]
```diff
@@ -65,145 +38,152 @@ export default class AddMovieComponent extends Component {
...
     return (
-      <>
-        <div className="addmovie-header">
-          {
-            editMode
+      <Formik
+        initialValues={movie}
+        validate={values => {
+          return this.isDataValid(values);
+        }}
```
Comment (alionaSipchuk) [inline src/components/AddMovie/AddMovieComponent.js:48]: incapsulate this logic into separate handler

File: src/components/AddMovie/AddMovieComponent.js [hunk 2]
```diff
@@ -65,145 +38,152 @@ export default class AddMovieComponent extends Component {
...
+        onSubmit={(values, { setSubmitting }) => {
+          values.runtime = Number(values.runtime);
```
Comment (alionaSipchuk) [inline src/components/AddMovie/AddMovieComponent.js:50]: why did you convert number value with Number constructor, if this value not a number pls do it averywhere!

File: src/components/AddMovie/AddMovieComponent.js [hunk 3]
```diff
@@ -65,145 +38,152 @@ export default class AddMovieComponent extends Component {
...
+          onSubmit(values);
+          setSubmitting(false);
```
Comment (alionaSipchuk) [inline src/components/AddMovie/AddMovieComponent.js:52]: ditto

**Review by alionaSipchuk — APPROVED** [review]
fix small fixes)

File: src/components/AddMovie/AddMovieComponent.js (hunk 2 above)
Comment (shivakcpm) [reply src/components/AddMovie/AddMovieComponent.js:50]: even input type number , when we read from input it is giving string (4 as '4') but api strictly accepting numbers

**Pull request merged**